src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
Again, per spec that desugars to:

```css
:matches(.blah p) span { ... }
```

Or, simplified (because the :matches() doesn't do anything useful there):

```css
.blah p span { ... }
```